docs/getting-started/guide/cardinality.md

```diff
@@ -0,0 +1,274 @@
+# Cardinality
+
+This page will describe what cardinality is, how to add it to your Gaffer graph and how to use it in Gaffer queries.
+
+## What is Cardinality?
+
+Cardinality represents the number of unique Edges that are connected to a given Entity, also known as the degree of an Entity.
+
+``` mermaid
+graph TD
+  1(1, cardinality=4) --> 2
+  1 --> 3
+  1 --> 4
+  1 --> 5
+  3(3, cardinality=2) --> 5
+  3 --> 5
+  3 --> 5
+  3 --> 5
+```
+
+For very large graphs, updating this number accurately would be very costly in compute. This is because for each new Edge that is added, we would have to check both connected Entities to see if they are already connected to the other Entity, and this could be costly for Entities with a high cardinality. Instead, Gaffer uses approximate cardinality making use of a [HyperLogLog Sketch](https://datasketches.apache.org/docs/HLL/HLL.html), which estimates the cardinality with relatively low error.
+
+## How to add cardinality to your graph
```

Comment:
This page refers to cardinality estimates as just cardinality. I think it would be good for it to mention every now and then that this is an estimate. It might also be good to very briefly explain how to add the real (not estimate) cardinality as well or why that can't be done.

Reply:
It is mentioned why it shouldn't be done: "For very large graphs, updating this number accurately would be very costly in compute. This is because for each new Edge that is added, we would have to check both connected Entities to see if they are already connected to the other Entity, and this could be costly for Entities with a high cardinality. Instead, Gaffer uses approximate cardinality..."

I am not even really sure it is possible right now in Gaffer to store the "real" cardinality in the same way due to lack of aggregators/serialisers for a Set of objects which only returns its length. Instead you would just store a set of vertices and then get the length. For this reason, approximate cardinality is always used in Gaffer and the Entity group is often just called "cardinality".